test/evergreen.yml

```diff
@@ -5852,6 +5860,7 @@ buildvariants:
   expansions:
     CMAKE_TOOLCHAIN_FILE: -DCMAKE_TOOLCHAIN_FILE=../cmake/toolchains/mongodbtoolchain_stable_clang.cmake
     CMAKE_BUILD_TYPE: -DCMAKE_BUILD_TYPE=MSan
+    CC_OPTIMIZE_LEVEL: -DCC_OPTIMIZE_LEVEL=-O0
```

Comment:
I would prefer if at least one of these `-O0` optimisation levels had some explanation for why (i.e. that Clang treats `-Og` as `-O1` and makes the test binary much more difficult to debug). Unless it's already in this file somewhere?

Reply:
-